src/commands/connection.ts

```diff
@@ -0,0 +1,68 @@
+import { SongQueue } from '@/utils/song-queue';
+import {
+  createAudioPlayer,
+  getVoiceConnection,
+  joinVoiceChannel,
+  VoiceConnectionStatus,
+} from '@discordjs/voice';
+import { CommandInteraction } from 'discord.js';
+import Container from 'typedi';
+
+const queue = Container.get(SongQueue);
+
+export function executeLeave(interaction: CommandInteraction) {
+  try {
+    getVoiceConnection(interaction.guildId).disconnect();
```

Comment:
Is disconnect async and should we await to make sure nothing happens?

Reply:
it doesn't return a promise lol

Reply:
![image](https://user-images.githubusercontent.com/13706500/134801111-c094b6f2-b58e-4221-b030-a9e9c507f602.png)